frontend/javascripts/admin/voxelytics/workflow_list_view.tsx

```diff
@@ -12,6 +12,18 @@ import { usePolling } from "libs/react_hooks";
 import { formatCountToDataAmountUnit, formatDateMedium, formatNumber } from "libs/format_utils";
 import Toast from "libs/toast";
 import { runStateToStatus, VX_POLLING_INTERVAL } from "./utils";
+import Persistence from "libs/persistence";
+import * as Utils from "libs/utils";
+import { PropTypes } from "@scalableminds/prop-types";
+
+const { Search } = Input;
+
+const persistence = new Persistence<Pick<{ searchQuery: string }, "searchQuery">>(
```

Comment:
tbh I just used this because it was used in other places aswell. I assume this is needed for `Utils.filterWithSearchQueryAND`?

Reply:
the persistence module ensures that refreshing the page or navigating-away-and-back will keep the search term intact :)